lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -977,4 +977,22 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
             binder.op, resultType, tensor, /*memory_format=*/noneVal);
         return success();
       });
+    
+  patterns.onOp("Mod", 13,
+                [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+                  Torch::ValueTensorType resultType;
+                  Value lhs, rhs;
+                  int64_t fmod;
+                  if (binder.tensorOperands(lhs, rhs) ||
+                      binder.tensorResultType(resultType) || 
+                      binder.s64IntegerAttr(fmod, "fmod", 0)) {
+                    return failure();
+                  }
+                  Value constFmod = rewriter.create<Torch::ConstantIntOp>(
+                		    binder.getLoc(), rewriter.getType<Torch::IntType>(),
+                		    rewriter.getIntegerAttr(rewriter.getIntegerType(64), fmod));
+                  rewriter.replaceOpWithNewOp<Torch::AtenFmodTensorOp>(
+                      binder.op, resultType, lhs, rhs, constFmod);
```

Comment:
Hi @pavs315, this piece of code is wrong. The `AtenFmodTensorOp` takes only two operands. Refer: https://github.com/llvm/torch-mlir/blob/89e02c195b910621246c55003fca86558162c6da/include/torch-mlir/Dialect/Torch/IR/GeneratedTorchOps.td#L11593-L11615
Here, you need to put a condition based on the value of `fmod` attribute. If it's 0 then you should create `AtenRemainderTensorOp` and if it's 1 then you should create `AtenFmodTensorOp`.